src/lib/articles.scss

```diff
@@ -0,0 +1,37 @@
+.articleEdit,
```

Comment:
using this as a stylesheet (`.scss`) will pollute the global stylesheet; are you sure you can't refactor this with componentization?

if not, I would instead inline this into a layout where you can [nest global styles as described in this SO post](https://stackoverflow.com/a/71329246/7589775).

Reply:
Fixed in cace9c544b397d30ebfce925a835e0ff9d6cccd1